### ISSUE TITLE
Solar Sensor connects to weather.com

### ISSUE THREAD

**Issue by Papermanzero** [mgba-emu/mgba#869]
Just a nice-to-have feature:
To get the data for the current weather, mgba could connect to weather.com. 
The user has only to enter the position and mgba is pulling the information regularly. 
The solar sensor for Boktai adjust according to the information (time, sunrise, sunset, UV index and weather status e.g. cloudy).

**Comment by endrift** [comment]
Once scripting is done you can do this with a script. This will not be a core feature.

**Comment by TsukiZero** [comment]
sorry for the necro, but I'd like to be notified of this when it is done.
